From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Fredric Silberberg <fredric@fsilberberg.com>
Date: Fri, 5 Dec 2014 14:42:54 -0500
Subject: [PATCH 0532/6262] Java-Installer

Added the java-installer to the tools-zip created by the plugins

Change-Id: Ibf03e3cf83c6c8a7663c4c55c0fb18623020039e
---
 eclipse-plugins/edu.wpi.first.wpilib.plugins.core/pom.xml | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/eclipse-plugins/edu.wpi.first.wpilib.plugins.core/pom.xml b/eclipse-plugins/edu.wpi.first.wpilib.plugins.core/pom.xml
index d322dce1f09db84b0be7e76227c6485c4842e16d..c273a930961b7c12a3489ee5a45d3f6412b9ad26 100644
--- a/eclipse-plugins/edu.wpi.first.wpilib.plugins.core/pom.xml
+++ b/eclipse-plugins/edu.wpi.first.wpilib.plugins.core/pom.xml
@@ -92,6 +92,13 @@
 				  <outputDirectory>${tools-zip}</outputDirectory>
 				  <destFileName>OutlineViewer.jar</destFileName>
 	            </artifactItem>
+		    <artifactItem>
+		      <groupId>edu.wpi.first.wpilib</groupId>
+		      <artifactId>java-installer</artifactId>
+		      <version>1.0-SNAPSHOT</version>
+		      <outputDirectory>${tools-zip}</outputDirectory>
+		      <destFileName>java-installer.jar</destFileName>
+		    </artifactItem>
               </artifactItems>
                 
               <overWriteReleases>false</overWriteReleases>
